python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -1099,7 +1099,9 @@ def _maybe_wrap_result(result: Any, func: Callable, /, *args, **kwargs) -> Any:
     """
     Wraps "result" in a fast-slow proxy if is a "proxiable" object.
     """
-    if _is_final_type(result):
+    if isinstance(result, (int, str, float, bool, type(None))):
+        return result
+    elif _is_final_type(result):
```

Comment:
Should we incorporate this check in `_is_final_type`?

Reply:
I don't think it is a good idea to add it to `_is_final_type`, those are for pandas types and the ones here we are trying to capture are native python types.